packages/core/src/routes/admin-user.ts

```diff
@@ -78,7 +78,7 @@ export default function adminUserRoutes<T extends AuthedRouter>(router: T) {
 
       const user = await findUserById(userId);
 
-      ctx.body = pick(user, ...userInfoSelectFields);
+      ctx.body = pick(user, 'roleNames', ...userInfoSelectFields);
```

Comment:
nit: since this is a temporary solution, maybe add a comment to locate the upcoming changes?

Reply:
No need for that, the word `roleNames` is enough (and is everywhere).